codice-itest-impl/src/main/java/org/codice/itest/SkipAwareExecutorService.java

```diff
@@ -0,0 +1,49 @@
+package org.codice.itest;
+
+import org.codice.itest.api.TestResult;
+import org.codice.itest.api.TestResultFactory;
+import org.codice.itest.config.ITestConfigurationProperties;
+import org.codice.itest.reporter.RemainingTestsReporter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.concurrent.*;
```

Comment:
❓ Can the Arrays dependency be removed? It doesn't seem to be used in this class.

✏️ Avoid * imports

Reply:
Ya I can,  and I can also add the codice headers